integration-tests/src/test/java/com/datastax/oss/driver/core/ConnectIT.java

```diff
@@ -131,6 +132,8 @@ public void should_cleanup_on_lbp_init_failure() {
     DriverConfigLoader loader =
         SessionUtils.configLoaderBuilder()
             .without(DefaultDriverOption.LOAD_BALANCING_LOCAL_DATACENTER)
+            .withClass(
```

Comment:
This would force the DSE driver to use the OSS LBP. I think instead we would like to make this test work with the DSE driver using its default LBP (DseLoadBalancingPolicy).

From what I see we could do this but the incriminated test is creating a hard-coded `CqlSession` line 137. This should be changed to `SessionUtils.baseBuilder()`.